webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/wkw/WKWFile.scala

```diff
@@ -0,0 +1,113 @@
+package com.scalableminds.webknossos.datastore.dataformats.wkw
+
+import java.io._
+import org.apache.commons.io.IOUtils
+import com.google.common.io.LittleEndianDataInputStream
+import com.scalableminds.util.tools.BoxImplicits
+import com.scalableminds.webknossos.datastore.dataformats.wkw.util.ResourceBox
+import net.jpountz.lz4.LZ4Factory
```

Comment:
Where does this actually come from now? Thought it came from wrap since it is not listed in the dependencies. Anyway, net to be careful with this one.

Reply:
It comes from `"org.lz4" % "lz4-java"`, which is a dependency of play. I now also added it as an explicit dependency in Dependencies.scala since we are importing it directly.